From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Thu, 25 Apr 2019 19:37:54 +0200
Subject: bash-completion: don't sort syslog priorities

By default, the available completions are sorted alphabetically, which
is counterproductive in case of syslog priorities. Override the default
behavior using the `nosort` option

(cherry picked from commit f8f542482e1df9b99144becf74dfae13a4d4403d)
---
 shell-completion/bash/journalctl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/shell-completion/bash/journalctl b/shell-completion/bash/journalctl
index 3beb347..51e1e6a 100644
--- a/shell-completion/bash/journalctl
+++ b/shell-completion/bash/journalctl
@@ -83,6 +83,7 @@ _journalctl() {
                         ;;
                         --priority|-p)
                                 comps=${__syslog_priorities[*]}
+                                compopt -o nosort
                         ;;
                         --unit|-u)
                                 comps=$(journalctl -F '_SYSTEMD_UNIT' 2>/dev/null)
